site/id/tutorials/keras/classification.ipynb

```diff
@@ -0,0 +1,1025 @@
+    "    plt.xlabel(class_names[train_labels[i]])\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "59veuiEZCaW4"
+   },
+   "source": [
+    "## Membangun model\n",
+    "\n",
+    "Untuk dapat membangun sebuah model neural network, perlu dilakukan konfigurasi layer untuk model, setelah itu kompilasi dari model tersebut."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "Gxg1XGm0eOBy"
+   },
+   "source": [
+    "### Pengaturan layer\n",
+    "\n",
+    "Komponen utama dari neural network adalah *layer*. Layer mengekstraksi representasi dari data yang masuk ke dalam layer-layer tersebut. Harapannya, representasi tersebut berguna untuk mengatasi permasalahan yang ingin diselesaikan.\n",
+    "\n",
+    "Kebanyakan deep learning terdiri atas penggabungan layer-layer sederhana. Kebanyakan layer, seperti `tf.keras.layers.Dense`, memiliki parameter-parameter yang dipelajari ketika proses training dijalankan"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "9ODch-OFCaW4"
+   },
+   "outputs": [],
+   "source": [
+    "model = keras.Sequential([\n",
+    "    keras.layers.Flatten(input_shape=(28, 28)),\n",
+    "    keras.layers.Dense(128, activation='relu'),\n",
+    "    keras.layers.Dense(10, activation='softmax')\n",
+    "])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "gut8A_7rCaW6"
+   },
+   "source": [
+    "Layer pertama dalam model neural network ini, `tf.keras.layers.Flatten`, melakukan transformasi dari format gambar yang mulanya aray dua dimensi (28 x 28 pixel) menjadi aray satu dimensi (28 * 28 = 784 pixel). Bayangkan layer ini sebagai layar yang membuat pixel-pixel dalam gambar menjadi satu baris. Layer ini tidak memiliki parameter untuk dipelajari; layer ini hanya mengubah format dari data.\n",
+    "\n",
+    "Setelah pixel diubah menjadi satu baris, model yang akan dibuat terdiri dari dua buah layer `tf.keras.layers.Dense`. Kedua layer ini terhubung secara penuh. Layer `Dense` yang pertama memiliki 128 node (atau neuron). Layer yang kedua (dan terakhir) memiliki 10 node *softmax* yang mengembalikan sebuah array dari 10 nilai probabilitas yang apabila dijumlahkan hasilnya adalah 1. Setiap node memiliki score yang mengindikasikan probabilitas bahwa gambar yang sedang diproses merupakan 1 dari 10 kelas label.\n",
+    "\n",
+    "### Proses kompilasi model\n",
+    "\n",
+    "Sebelum model siap untuk di training, model ini memerlukan beberapa pengaturan tambahan. Pengaturan ini ditambahkan ketika proses kompilasi:\n",
+    "\n",
+    "* *loss function* —Fungsi ini menghitung seberapa akurat model ketika proses training dilakukan. Anda ingin meminimalisir nilai dari fungsi ini untuk \"mengarahkan\" model ke arah yang benar.\n",
```

Comment:
> Anda ingin meminimalisir nilai dari fungsi ini

should it be something like 

> Anda ingin meminimalisir nilai kerugian dari fungsi ini

Reply:
Okay, I will try to try edit those text. Thank you!

Reply:
I edited the text according to your suggestion.